File: sdks/dotnet/Acli.Spec/OutputFormat.cs
namespace Acli;

public enum OutputFormat
{
    Text,
    Json,
    Table,
}

public static class OutputFormatParser
{
    public static OutputFormat Parse(string? s) => s?.ToLowerInvariant() switch
    {
        "json" => OutputFormat.Json,
        "table" => OutputFormat.Table,
        _ => OutputFormat.Text,
    };
}

File: sdks/java/src/main/java/dev/acli/OutputFormat.java
package dev.acli;

/** Supported output formats per ACLI spec §2.1. */
public enum OutputFormat {
    text,
    json,
    table;

    public static OutputFormat parse(String s) {
        if (s == null) {
            return text;
        }
        return switch (s.toLowerCase()) {
            case "json" -> json;
            case "table" -> table;
            default -> text;
        };
    }
}
